v3/src/components/graph/adornments/adornment-content-info.ts

```diff
@@ -23,5 +25,10 @@ export function getAdornmentContentModels() {
 }
 
 export function getAdornmentTypes() {
-  return Object.values(gAdornmentContentInfoMap).map(info => info.type)
+  return Object.values(gAdornmentContentInfoMap).map((info) => {
+    return {
+      parentType: info.parentType,
+      type: info.type,
+    }
+  })
```

Comment:
FYI, you should be able to replace this with:
```suggestion
  return Object.values(gAdornmentContentInfoMap).map(({ parentType, type }) => ({ parentType, type }))
```